.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,268 @@
+name: Build and test
+
+on:
+  push:
+    branches:
+      - master
+      - "*-rc"
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    branches:
+      - "**"
+
```

Comment:
Can you please specify the workflow permissions?
Should be 
```
permissions:
  contents: read
```

Reply:
addressed!